### PR TITLE
[stable/metrics-server] Update 'metrics-server' to v0.3.3.

### PR DESCRIPTION
**Pull request by bmcustodio**
### What this PR does / why we need it:

This PR updates `metrics-server` to v0.3.3.

#### Which issue this PR fixes

N/A

#### Special notes for your reviewer:

N/A

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/metrics-server/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.2
 description: Metrics Server is a cluster-wide aggregator of resource usage data.
 name: metrics-server
-version: 2.8.2
+version: 2.8.3
```
Comment (olemarkus) [inline stable/metrics-server/Chart.yaml:5]: Bump minor version here

File: stable/metrics-server/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.3.2
```
Comment (olemarkus) [inline stable/metrics-server/Chart.yaml:2]: Bump version here too

**Review by olemarkus — CHANGES_REQUESTED** [review]

**Comment by PierluigiLenociAkelius** [comment]
This PR is uncomplete and now is useless. Ref: #16618
@bmcstdio could you please close it?

**Comment by PierluigiLenociAkelius** [comment]
https://github.com/helm/charts/pull/16656

**Comment by PierluigiLenociAkelius** [comment]
@bmcstdio could you please close this?

**Pull request closed without merging**